autoscheduler/scraper/management/commands/utils/scraper_utils.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+from typing import List
+from itertools import product
+
+def get_all_terms() -> List[str]:
+    """ Generates all of the terms, from 2019 until now """

```

Comment:
The reason we need to actually get all of them is since `scrape_grades` needs a Section model to attach the Grade models to, otherwise the `scrape_grades` will fail for that given term. 

However, we could make an option for the `all` to make it only scrape a given year / location / semester. I.e. `scrape_courses -c 1` would scrape all of the courses for all years & semesters only for college station, or `scrape_courses -y 2019` would scrape all of the semesters/locations for 2019. (Basically like what we have for `scrape_grades`)